compiler/passes/insertWideReferences.cpp

```diff
@@ -198,8 +198,12 @@
   #define DEBUG_PRINTF(...)
 #endif
 
-static void debug(Symbol* sym, const char* format, ...) {
+static void debug(BaseAST* base, const char* format, ...) {
 #ifdef PRINT_WIDE_ANALYSIS
+  Symbol* sym = toSymbol(base);
+  if (!sym) {
```

Comment:
minor, but `!sym` => `sym != NULL` to match the style of the `sym == NULL` below?